src/modules/user/reducer.js

```diff
@@ -6,24 +6,22 @@ import { createReducer } from '_utils/redux'
 
 import { LOGIN } from '../authentication/actions'
 
-import { GET_USER } from './actions'
+import { GET_USER, UPDATE_USER } from './actions'
 
 const INITIAL_STATE = new User()
 
 const saveUserData = payload => {
-  const { accessToken, ...data } = humps.camelizeKeys(payload)
-  cookies.save('accessToken', accessToken, { path: '/', maxAge: 60 * 60 * 24 * 6 })
-  // 60 * 60 * 24 * 6 expires in the next 6 days
+  const { ...data } = humps.camelizeKeys(payload)
   cookies.save('user', data, { path: '/', maxAge: 60 * 60 * 24 * 6 })
   return new User({
     ...data,
-    accessToken,
   })
 }
 
 const user = createReducer(INITIAL_STATE, {
   [LOGIN.FULFILLED]: (state, { payload }) =>
     saveUserData({ ...payload.user, accessToken: payload.key }),
   [GET_USER.FULFILLED]: (state, { payload }) => saveUserData({ ...payload[0] }),
+  [UPDATE_USER.FULFILLED]: (state, { payload }) => saveUserData({ ...payload }),
```

Comment:
you don't have to use an object and spread operator to pass the payload here, you can just to `saveUserData(payload)`